lib/bolt/cli.rb

```diff
@@ -362,6 +379,31 @@ def run_plan(plan_name, plan_arguments, nodes, options)
       result.ok? ? 0 : 1
     end
 
+    def apply_manifest(code, targets, filename = nil)
+      begin
+        ast = Puppet::Pops::Parser::EvaluatingParser.new.parse_string(code, filename)
```

Comment:
It looks like we still need something to pull this into scope. When I run `bolt apply test.pp -n localhost` I get
```
NameError: uninitialized constant Bolt::CLI::Puppet
  /Users/michaelsmith/puppetlabs/bolt/lib/bolt/cli.rb:386:in `rescue in apply_manifest'
  /Users/michaelsmith/puppetlabs/bolt/lib/bolt/cli.rb:383:in `apply_manifest'
  /Users/michaelsmith/puppetlabs/bolt/lib/bolt/cli.rb:275:in `execute'
  /Users/michaelsmith/puppetlabs/bolt/exe/bolt:10:in `<top (required)>'
  /Users/michaelsmith/puppetlabs/bolt/.bundle/gems/ruby/2.5.0/bin/bolt:23:in `load'
  /Users/michaelsmith/puppetlabs/bolt/.bundle/gems/ruby/2.5.0/bin/bolt:23:in `<top (required)>'
```

Reply:
Probably because we call this before `pal`.

Reply:
Whoops so that `require 'bolt/catalog'` I just removed was necessary after all. :(

Reply:
Or something like it. Calling `pal` before the block also works.

Reply:
Maybe worth throwing this into a helper in Bolt::PAL?

Reply:
"load-bearing", shall we say